### PR TITLE
Layout clash in Audio Recordings on the Home page

### PR DESCRIPTION
**Pull request by Joydev**
audio list layout clash in homepage

### CODE REVIEW

File: frontend/src/index.html [hunk 1]
```diff
@@ -86,10 +86,10 @@ <h4 class="text-sm font-semibold uppercase md:text-base">Audio Recordings</h4>
     <div class="mt-4 grid grid-cols-1 gap-2 md:grid-cols-2">
       {% for audio in audios offset: 25 limit: 8 %}
         <a href="./audios/{{ audio.objectID }}/">
-          <div class="grid  grid-cols-6 rounded-md border border-indigo-300 hover:!bg-yellow-100 hover:!bg-opacity-50 md:grid-cols-8">
-            <div class="col-span-1 ml-2">
+          <div class="flex items-center rounded-md border border-indigo-300 hover:!bg-yellow-100 hover:!bg-opacity-50">
+            <div class="mx-2 mt-1.5">
               <button
-                class="mt-2 w-8 flex-none rounded-full bg-neutral-600 fill-current focus:outline-none"
+                class="w-8 flex-none rounded-full bg-neutral-600 fill-current focus:outline-none"
```
Comment (anantakrishna) [inline frontend/src/index.html:92]: Here we have `svg` inside `button` inside `div`. I believe we can eliminate this hierarchy and use only `svg`. See the same block in search results. Simplify the markup.

File: frontend/src/index.html [hunk 2]
```diff
@@ -86,10 +86,10 @@ <h4 class="text-sm font-semibold uppercase md:text-base">Audio Recordings</h4>
     <div class="mt-4 grid grid-cols-1 gap-2 md:grid-cols-2">
       {% for audio in audios offset: 25 limit: 8 %}
         <a href="./audios/{{ audio.objectID }}/">
-          <div class="grid  grid-cols-6 rounded-md border border-indigo-300 hover:!bg-yellow-100 hover:!bg-opacity-50 md:grid-cols-8">
-            <div class="col-span-1 ml-2">
+          <div class="flex items-center rounded-md border border-indigo-300 hover:!bg-yellow-100 hover:!bg-opacity-50">
```
Comment (anantakrishna) [inline frontend/src/index.html:89]: - We don't use important specifiers.
- Remove hover, it's too flashy.
- Why `indigo`? We don't use this color elsewhere.
- This `div` is also redundant, apply styles directly to the `a` element.

File: frontend/src/index.html [hunk 3]
```diff
@@ -106,9 +106,9 @@ <h4 class="text-sm font-semibold uppercase md:text-base">Audio Recordings</h4>
                 </svg>
               </button>
             </div>
-            <div class="col-span-5 content-center pt-1 md:col-span-7">
+            <div class="mr-2 mt-1 content-center overflow-hidden">
```
Comment (anantakrishna) [inline frontend/src/index.html:109]: - Vertical gap can be better and more consistently implemented as a padding of the parent element.
- On the right, it's better to be a padding.

**Review by anantakrishna — CHANGES_REQUESTED** [review]

File: frontend/src/index.html (hunk 2 above)
Comment (Joydev) [reply frontend/src/index.html:89]: updated as per review....

File: frontend/src/index.html (hunk 3 above)
Comment (Joydev) [reply frontend/src/index.html]: updated ...

File: frontend/src/index.html (hunk 1 above)
Comment (anantakrishna) [reply frontend/src/index.html]: Not fully. I proposed getting rid of the `button` altogether. Keep only `svg` here.

**Review by anantakrishna — CHANGES_REQUESTED** [review]
Remove `button` altogether. Keep only `svg` here.

**Review by anantakrishna — APPROVED** [review]

**Pull request merged**